requirements.txt

```diff
@@ -29,3 +29,4 @@ jsonref==0.1
 lxml==4.9.1
 openpyxl==3.0.10
 schema==0.4.0
+setuptools==65.5.0
```

Comment:
This may have been auto added, we don't normally bundle the core python libraries `setuptools` here as it is python version specific.  Suggest removing this from the commit.

Reply:
I think this was actually added deliberately, as the builds were previously failing due to `setuptools` not being included in the rtd build environment anymore.

* https://readthedocs.org/projects/threesixtygiving-standard/builds/25462590/